dodola/tests/test_repository.py

```diff
@@ -1,15 +1,16 @@
-from xarray import Dataset
-from dodola.repository import memory_repository
+from xarray import Dataset, open_zarr
+import dodola.repository
 
 
 def test_memory_repository_read():
     """Basic test that memory_repository.read() works"""
-    storage = memory_repository(storage={"foo": Dataset({"bar": 123})})
-    assert storage.read(url_or_path="foo") == Dataset({"bar": 123})
+    url = "memory://test_memory_repository_read.zarr"
+    Dataset({"bar": 321}).to_zarr(url)  # Manually write to memory FS.
```

Comment:
Interesting! Thought you had to import `Dataset` from `xarray.core`.

Reply:
That's where it might be defined etc but they import it in `xarray.__init__.py` from `xarray.core` for a nice interface kinda like `np.array` and `pd.DataFrame` you have `xr.Dataset` and `xr.DataArray`!